integrations/standalone/tests/integration/parts/permissions.ts

```diff
@@ -4,27 +4,38 @@ import { Checkbox } from '../../pageobjects/Checkbox';
 
 class Permissions extends PartObject {
   viewable: Checkbox;
+  defaultIsChecked: boolean;
 
-  constructor(part: Part) {
+  constructor(part: Part, defaultIsChecked: boolean) {
     super(part);
     this.viewable = part.checkbox('Allow all workflow users to view the process on the Engine');
+    this.defaultIsChecked = defaultIsChecked;
   }
 
   async fill() {
     await this.viewable.click();
   }
 
   async assertFill() {
-    await this.viewable.expectUnchecked();
+    if (this.defaultIsChecked) {
+      await this.viewable.expectUnchecked();
+    } else {
+      await this.viewable.expectChecked();
+    }
   }
 
   async clear() {
     await this.viewable.click();
   }
 
   async assertClear() {
-    await this.viewable.expectChecked();
+    if (this.defaultIsChecked) {
+      await this.viewable.expectChecked();
+    } else {
+      await this.viewable.expectUnchecked();
+    }
   }
 }
 
-export const PermissionsTest = new NewPartTest('Permissions', (part: Part) => new Permissions(part));
+export const PermissionsTest = (defaultIsChecked: boolean = true) =>

```

Comment:
if the default value is good enough you don't need to define the type: `(defaultIsChecked = true)`